### ISSUE TITLE
[feature request] Provide casks from binary-factory.kde.org

### ISSUE THREAD

**Issue by forivall** [KDE-mac/homebrew-kde#340]
https://binary-factory.kde.org/view/MacOS/ provides dmg's for a number of kde packages. It would be great to provide them as an alternative to the source packages that this tap provides (idk if this tap provides bottles, last time i installed something it built it from source)

**Comment by wrobelda** [comment]
Brew provides such functionality through bottles: https://docs.brew.sh/Bottles

Duplicate of #163

**Comment by forivall** [comment]
Yes, they are provided through bottles, but they are also available through [casks](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap#casks), which, imo, is more ideal for UI apps.

**Comment by wrobelda** [comment]
Ah, my bad, I misunderstood your request. Yeah, this could also be totally automated.
